mdns.lua

```diff
@@ -41,10 +41,7 @@
 ]]--
 
 local mdns = {}
-
 local socket = require('socket')
-
-
```

Comment:
Ideally there are line breaks between non-associated blocks to help readability. However, I wasn't the original author, so I am less picky

Were the inline if/returns and whitespace changes done with a code formatter or linter?

Reply:
No......want me to undo?

Reply:
I would appreciate it. Happy to commit the specific improvements you made, but a code-style refactor + code change is harder to validate and approve.

Reply:
Argh...sorry I am crap with git.....I'm sure I could have made it easier, but anyway I have removed the line spaces etc
Hope this is better